src/System.IO.FileSystem/src/System/IO/Win32FileSystemEnumerable.cs

```diff
@@ -398,7 +399,7 @@ public override bool MoveNext()
         [SecurityCritical]
```

Comment:
we should remove these from this assembly at some point ... https://github.com/dotnet/corefx/issues/12592